Move content from community repo (#4)
open-telemetry/sig-end-user: end-user-interviews/README.md

# <img src="https://opentelemetry.io/img/logos/opentelemetry-logo-nav.png" alt="OpenTelemetry Icon" width="45" height=""> OpenTelemetry community End User interview FAQ

## What are end user interviews? 

> - The OpenTelemetry community conducts end user interviews to gather high fidelity information from users though the medium of synchronous discussions to learn about their challenges, preferences, and behaviors regarding OpenTelemetry. 

Reviewer: ```suggestion
- The OpenTelemetry community conducts end user interviews to gather high fidelity information from users though synchronous discussions to learn about their challenges, preferences, and behaviors regarding OpenTelemetry. 
```

## Why do these interviews happen? 

> - Many vendors in this space conduct interviews with their customers but that feedback doesn't always make its way back to the community directly. These interviews will help us test our assumptions, categorize end-users, and describe their needs so that the community can react to their feedback. 

Reviewer: ```suggestion
- These interviews will help us test our assumptions, categorize end-users, and describe their needs so that the community can react to their feedback. 
```

## When do these sessions happen? 

> - Our end user working group goal is to hold at least one session a month. 

Reviewer: ```suggestion
- Our End-User SIG goal is to hold at least one session a month. 
```

> - Our scheduling preference is for these sessions to occur at 10:00 AM PST on Thursdays, however the final schedule will be dictated by the end-user interviewee’s availability.

Reviewer: ```suggestion
- Our scheduling preference is for these sessions to occur at 10:00 PST/13:00 ET/19:00 CET on Thursdays, however the final schedule will be dictated by the end-user interviewee’s availability.
```

## Are the interview sessions recorded? 

> - Sessions are only recorded if the end user is comfortable with them being recorded. In this directory, you can find links to both video recordings and session summaries created by the Facilitator and Helper roles, as appropriate. 

Reviewer: ```suggestion
- Sessions are only recorded if the end user is comfortable with them being recorded. In this directory, you can find links to both video recordings and session summaries created by the Facilitator and Helper roles, as appropriate. Video recordings are also available on our [YouTube Channel](https://youtube.com/playlist?list=PLVYDBkQ1TdywIl9xKEo5_u7zlwY38dW43&si=lQ64z2uS9V59bPE4)
```

  - Interviewee selection criteria vary and change depending on the needs of the community. Some example criteria includes “people who have used OTel before”, “people who have implemented monitoring, telemetry, observability either in code or in platforms”, “people who develop software and don’t feel they have enough information about how it works in environments other than their laptop”

* ### OTel Community Helper: - Required
  - The OTel community helper is a named role for each session. The community helper will complete coordination and amplification tasks such as tracking session recording links, acting as note keeper, and publishing up an interview summary. The community helper can also be the same person as the interview facilitator. 

* ### OTel Community member audience: - Optional
  - Members from the OTel community are welcome to observe the interview and coordinate with the facilitator if there are any specific topics they’d like to be included during the interview. 

## How can I get involved or learn more?

> - Are you interested in providing the community your feedback in this way? Please contact us in [#otel-user-research](https://cloud-native.slack.com/archives/C01RT3MSWGZ) on CNCF Slack to talk about scheduling. And as always, you can provide your feedback to the community in many ways including our community surveys too. 

Reviewer: ```suggestion
- Are you interested in providing the community your feedback in this way? Please contact us in [#otel-user-research](https://cloud-native.slack.com/archives/C01RT3MSWGZ) on CNCF Slack to talk about scheduling. And as always, you can provide your feedback to the community in many ways, including our community surveys too. 
```
